src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpMessage.cs

```diff
@@ -23,8 +23,8 @@ public abstract partial class HttpMessage  : IDisposable
 
         public abstract void AddHeader(HttpHeader header);
 
-        public virtual void AddHeader(string name, string value)
-            => AddHeader(new HttpHeader(name, value));
+        public virtual void AddHeader(string name, HeaderValues values)
```

Comment:
I'd suggest that using  `params`  to allow for multiple header values would make the API more friendly and convenient.   Wrapping them into `HeaderValues` or another construct may be better left as a responsibility of the `HttpHeader` class.